source/includes/Searching/Full/_intro.md

```diff
@@ -18,9 +18,19 @@ curl
 ```
 > Go [here](#full-searches_request-table) to se a full example of a request body and [here](#full-searches_response) to se a response body.
 
+<h5>EU CLUSTER:</h5>
```

Comment:
The whole EU/US cluster thing is a very internal thing, and not something our customers are aware of and nor should they be. Instead I would say that we should just word it as customers being in either the US or the rest of the world.